masterlist.yaml

```
@@ -6644,27 +6644,72 @@ plugins:
       - crc: 0xA0C60505
         util: 'TES4Edit v4.0.3'
 
+  - name: '(Kragenir''s Death Quest|KDQ - Rural Line Additions|KragenirsDeathQuest-LetThePeopleDrink patch)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/26219/'
+        name: 'Kragenirs Death Quest'
   - name: 'Kragenir''s Death Quest.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/26219/' ]
+    req: [ *SilentVoice ]
+    inc:
+      - 'Servant of the Dawn.esp'
+      - '300_White Stallion 4.esp'
```

These should be sorted alphanumerically.

```suggestion
      - '300_White Stallion 4.esp'
      - 'Servant of the Dawn.esp'
```